pyhive/tests/test_hive.py

```diff
@@ -139,3 +139,30 @@ def test_no_result_set(self, cursor):
         cursor.execute('USE default')
         self.assertIsNone(cursor.description)
         self.assertRaises(hive.ProgrammingError, cursor.fetchone)
+
+
+@unittest.skipIf(sys.version_info.major == 3, 'Hive not yet supported on Python 3')
+class TestHiveAuth(unittest.TestCase):
+    __test__ = True
+
+    def test_ldap_connection(self):
+        import subprocess
+        import os
+        rootdir = os.environ['TRAVIS_BUILD_DIR']
+        subprocess.call(['sudo', 'cp', rootdir + '/scripts/travis-conf/hive/hive-site-ldap.xml',
```

Comment:
- Prefer subprocess.check_call, which throws an exception if the command fails
- Prefer os.path.join(rootdir, 'scripts', 'travis-conf', ...)